apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClientBuilder.cs

```diff
@@ -42,7 +42,16 @@ public new EmulatorDetection EmulatorDetection
         partial void InterceptBuildAsync(CancellationToken cancellationToken, ref Task<SpannerClient> task) =>
             task = MaybeCreateEmulatorClientBuilder()?.BuildAsync(cancellationToken);
 
-        private SpannerClientBuilder MaybeCreateEmulatorClientBuilder()
+        /// <summary>
+        /// May return a builder that will connect to the emulator under certain conditions.
+        /// </summary>
+        /// <remarks>
+        /// Returns null if <see cref="EmulatorDetection"/> is <see cref="EmulatorDetection.None"/>
+        /// or <see cref="EmulatorDetection.ProductionOnly"/>. Otherwise, returns a builder to
```

Comment:
I'd write this as "it returns null if the emulator should not be used" - it will return null if EmulatorDetection is EmulatorOrProduction but the environment variable isn't configured, for example.